### PR TITLE
enhance API: Clear trackers

### PR DESCRIPTION
**Pull request by ajax16384**
allow clear torrent_info trackers list

### CODE REVIEW

File: include/libtorrent/torrent_info.hpp [hunk 1]
```diff
@@ -268,6 +268,7 @@ namespace libtorrent {
 		// number. For more information, see announce_entry_.
 		void add_tracker(std::string const& url, int tier = 0);
 		std::vector<announce_entry> const& trackers() const { return m_urls; }
+		void clear_trackers();
```
Comment (arvidn) [inline include/libtorrent/torrent_info.hpp]: could you include `clear_trackers()` in the documentation above as well?

File: src/torrent_info.cpp [hunk 2]
```diff
@@ -1525,6 +1525,10 @@ namespace {
 			{ return lhs.tier < rhs.tier; });
 	}
 
+	void torrent_info::clear_trackers() {
```
Comment (arvidn) [inline src/torrent_info.cpp]: please put the open brace on the next line

**Comment by arvidn** [comment]
could you please add a unit test for `clear_trackers()` to `test/test_torrent_info.cpp` too?

File: include/libtorrent/torrent_info.hpp (hunk 1 above)
Comment (ajax16384) [reply include/libtorrent/torrent_info.hpp]: Ok

File: src/torrent_info.cpp (hunk 2 above)
Comment (ajax16384) [reply src/torrent_info.cpp]: Ok

**Comment by ajax16384** [comment]
Ok

**Pull request merged**